internal/receiver/prometheusremotewritereceiver/README.md

```diff
@@ -10,15 +10,17 @@ As of this writing, no official specification exists for remote write endpoints,
 
 As such, this receiver implements a best-effort mapping between such metrics.  If you find your use case or access patterns do not jive well with this receiver, please [cut an issue](https://github.com/signalfx/splunk-otel-collector/issues/new) to our repo with the specific data incongruity that you're experiencing, and we will do our best to provide for you within maintainable reason.
 
+Currently, we *only* support `sfx_gateway_compatibility` mode.  Trying to write a histogram, quantile, and anything other than a simple gauge or counter will fail if this configuration option is not enabled.  See [Receiver Configuration](#receiver-configuration) and the [Gateway Compatibility](#signalfx-gateway-compatibility-mode) for more details.
```

Comment:
If this statement is referring to the state of this PR there's no need to provide it since the component is a wip and isn't included in the distro as of yet. I think it creates more confusion than necessary.